### PR TITLE
Supervisor show course subject and finish

### PR DESCRIPTION
**Pull request by vietanhbkaaa**

### CODE REVIEW

File: app/views/rails_admin/main/start.js.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+$(".btn-actions-<%= @course_subject.id %>").html("<%= escape_javascript(render('btn_show_course_subject')) %>");
```
Comment (hienbx94) [inline app/views/rails_admin/main/start.js.erb]: remove ()

File: lib/rails_admin/finish_course_subject.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        register_instance_option :custom_key do
+          key
+        end
+
+        register_instance_option :controller do
+          proc do
+            @course_subject = CourseSubject.find params[:course_subject_id]
+            if @course_subject.update_attributes active: 2
+              flash[:success] = t "flashs.messages.successfully"
```
Comment (hienbx94) [inline lib/rails_admin/finish_course_subject.rb]: global flash

File: app/views/rails_admin/main/show_course.html.erb [hunk 3]
```diff
@@ -50,5 +54,26 @@
...
+    <div id="subject" class="tab-pane fade">
+    <div class="row">
+      <% @course_subjects.each do |course_subject| %>
+        <div class="form-group">
+          <div class="col-md-6">
+            <%= course_subject.subject.name %>
+          </div>
+          <div class="col-md-6 btn-actions-<%= course_subject.id %>">
+            <% if course_subject.init? %>
+              <%= link_to t("admin.actions.show_course_subject.start"), start_course_subject_path(course_subject_id: course_subject.id), class: "btn btn-primary", remote: true %>
```
Comment (hellovietnam93) [inline app/views/rails_admin/main/show_course.html.erb]: 2 lines

**Comment by binhthanhnguyen** [comment]
okay

**Comment by trantube94** [comment]
OK

**Pull request merged**